.github/workflows/ci.yml

```diff
@@ -89,6 +87,8 @@ jobs:
             github.event.pull_request.base.repo.full_name
           || !startsWith(github.head_ref, 'renovate/')
         )
+    env:
+      NX_CLOUD_ACCESS_TOKEN: ${{ secrets.NX_CLOUD_ACCESS_TOKEN_READ_ONLY }}
```

Comment:
This job runs Nx tasks on untrusted code, so Nx should be restricted to read-only access to the cache.